flowmachine/flowmachine/core/server/query_schemas/custom_fields.py

```diff
@@ -11,7 +11,9 @@
 
 class EventTypes(fields.List):
     """
-    A string representing an event type, for example "calls", "sms", "mds", "topups".
+    A list of strings representing an event type, for example "calls", "sms", "mds", "topups".
+
+    When deserialised, will be uniqueified, and prefixed with "events."
```

Comment:
What do you mean by "uniqueified"?

(And where is this actually happening - in the call to `super()`?)